content/luau/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: 'Conditionals'
+Description: 'Conditionals execute a code block depending on whether an expression is true.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Conditionals'
+  - 'Control Flow'
+  - 'Logic'
+  - 'Logical'
+CatalogContent:
+  - 'learn-lua'
+  - 'introduction-to-game-development'
+---
+
+A **conditional** determines whether or not certain blocks of code will run. A conditional checks the value of an expression, called the **condition**. If the condition is equal to `false`, then the code block within the conditional will be skipped. Otherwise, the code block will be run.
+
+Conditionals are used to control the flow of the execution of code within a program.
+
+## Basic Conditionals
+
+A basic conditional in Luau is given as follows:
+
+```psuedo
+if <condition> then
+  <code block>
+end
+```
+
+Note that, as seen above, `end` must be used to close out a conditional in Luau.
+
+### Examples of basic conditionals
+
+```pseudo
```

Comment:
```suggestion
```lua
```

Reply:
Should Lua formatting be used for Luau as well?

Reply:
Luau itself is not currently supported by markdown blocks, but using Lua will provide the correct text highlighting